test/src/811-commit-gateway/main

```diff
@@ -0,0 +1,23 @@
+cvmfs_test_name="Commit command in a repository connected to the gateway"
+cvmfs_test_autofs_on_startup=false
+cvmfs_test_suites="quick"
+
+cvmfs_run_test() {
+    set_up_repository_gateway || return 1
+
+    echo "*** prepare config file"
+    if !(grep -q "CVMFS_SERVER_URL" /etc/cvmfs/config.d/test.repo.org.conf); then
+        sudo bash -c "echo CVMFS_SERVER_URL=http://localhost/cvmfs/test.repo.org >> /etc/cvmfs/config.d/test.repo.org.conf"
+    fi
+    if !(grep -q "CVMFS_HTTP_PROXY" /etc/cvmfs/config.d/test.repo.org.conf); then
+        sudo bash -c "echo CVMFS_HTTP_PROXY="DIRECT" >> /etc/cvmfs/config.d/test.repo.org.conf"
+    fi
+    
+    echo "*** checking that enter, commit and exit succeed"
+    cvmfs_server enter test.repo.org -- false && return 50
+    cvmfs_server enter test.repo.org --transaction --repo-config /etc/cvmfs/repositories.d -- false && return 60
+    cvmfs_server enter test.repo.org --transaction --repo-config /etc/cvmfs/repositories.d -- cvmfs_server abort -f test.repo.org && return 70
```

Comment:
Shouldn't this and the following command succeed, i.e. `|| return 70` and `|| return 80`?